.github/workflows/tag-nautobot-app-v2.yml

```diff
@@ -38,5 +38,9 @@ jobs:
           python -m ntc_cookie_drift_manager \
             rebake \
             --push \
+            --post-action=ruff \
+            --post-action=poetry \
+            --disable-post-actions=black \
```

Comment:
It feels like we shouldn't make this the default to enable/disable actions, but perhaps it is a necessary override until each of the baked app `.cookiecutter.json` is updated accordingly (for example https://github.com/nautobot/nautobot-app-dev-example/blob/5577fcc9dc77b6dd8067c9a3e99773cf27a8006e/.cookiecutter.json#L28)

It seems this list of post actions gets set at first bake time? Ref: https://github.com/nautobot/cookiecutter-nautobot-app-drift-manager?tab=readme-ov-file#post-bake-actions

Reply:
This was Jan's suggestion instead of manually changing all the .cookiecutter.json files. After the rebake, they will all show the updated post actions in the .cookiecutter.json.

Reply:
I think it's fine to do this since it only runs on the static list of apps above

Reply:
Oh if it updates the json after each rebake, then well done Jan :) Worth clarifying in the DM docs that this is the case.

We will eventually remember to remove the lines again.